openlibrary/core/helpers.py

```diff
@@ -111,6 +111,11 @@ def safekey(x):
         return (k.__class__.__name__, k)
     return sorted(iterable, key=safekey, reverse=reverse)
 
+def days_since(then, now=None):
```

Comment:
Presumably extra line required above as well? Noting still a lint issue.

```suggestion

def days_since(then, now=None):
```